erpnext/accounts/doctype/pos_closing_entry/pos_closing_entry.py

```
@@ -441,3 +441,23 @@ def build_invoice_query(invoice_doctype, user, pos_profile, start, end):
 		)
 
 	return query
+
+
+@frappe.whitelist()
+@frappe.validate_and_sanitize_search_inputs
+def get_pos_profiles(doctype: str, txt: str, searchfield: str, start: int, page_len: int, filters: dict):
```

Ruff (0.14.14)

[warning] Unused function argument: `doctype`

(ARG001)